projects/kit/components/pdf-viewer/pdf-viewer.template.html

```diff
@@ -30,3 +30,18 @@ <h2 class="t-title">{{context.label}}</h2>
         ></iframe>
     </ng-template>
 </section>
+
+<div
+    *ngIf="emptyPreview"
+    class="t-error-message"
+    @tuiFadeIn
+>
+    <ng-container *ngIf="context.errorMessage; else defaultMessage">
+        {{context.errorMessage}}
+    </ng-container>
+    <ng-template #defaultMessage>
+        The PDF could not be loaded, there may be a network problem, or your
```

Comment:
Let's put it inside `@taiga-ui/i18n`:
https://github.com/Tinkoff/taiga-ui/blob/7b303d12621a7270e6a2b9d00651b76739486274/projects/i18n/interfaces/language.ts#L23

For example, property `unavailablePdfError` (or something shorter)